src/OctoshiftCLI.Tests/AdoApiTests.cs

```diff
@@ -37,5 +42,701 @@ public async void Get_User_Id_Invalid_Response()
             using var sut = new AdoApi(mockClient.Object);
             await Assert.ThrowsAsync<InvalidDataException>(async () => await sut.GetUserId());
         }
+
+        [Fact]
+        public async Task GetOrganizations()
```

Comment:
nit: Generally speaking I would use more descriptive names for test methods that exactly indicates what we expect. for example, `GetOrganizations_Should_Return_All_Orgs`. I know that sometimes it is obvious what the test is doing though.

Reply:
done